gateway/api/access_policies/jobs.py

```diff
@@ -0,0 +1,67 @@
+"""
+Access policies implementation for Job access
+"""
+import logging
+from django.contrib.auth import get_user_model
+from api.models import Job
+
+User = get_user_model()
```

Comment:
I see that we are only using `User` as a type. The problem is that I'm receiving a warning in VSCode when we use it:

```
Variable not allowed in type expressionPylance[reportInvalidTypeForm]
(variable) User: Any
```

How we cannot import directly the user model due to [E5142](https://github.com/pylint-dev/pylint-django/issues/278) error until we find a workaround my recommendation would be just remove it by now.